t/ui/15-admin-workers.t

```diff
@@ -151,6 +155,10 @@ subtest 'delete offline worker' => sub {
 };
 
 $driver->find_element('tr#worker_1 .worker a')->click();
+for (0 .. 3) {
+    last if $driver->get_title() eq 'openQA: Worker localhost:1';
+    sleep 1;
```

Comment:
I'm wondering why the sleep is required here. At least state in a comment that we had test runs where it otherwise fails.

Reply:
Without sleep there is a chance that whole loop completes too fast. The whole loop is here just making sure that next test will pass, I am not sure if additional comment is needed.